x/ccv/child/keeper/genesis.go

```diff
@@ -46,6 +76,8 @@ func (k Keeper) InitGenesis(ctx sdk.Context, state *types.GenesisState) {
 			k.SetUnbondingPacket(ctx, us.Sequence, us.UnbondingPacket)
 		}
 	}
+
+	return state.InitialValSet
```

Comment:
Am I correct in assuming that this is exactly the same as was included in the genesis.json, and is already in the format that chain startup needs?

Reply:
Correct. This is assuming the genesis file just includes the `[]abci.ValidatorUpdate` which is exactly what is expected by Tendermint